dockerfiles/run-gui-tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Just in case it's running, we stop the web server.
+docker-compose stop web
+
+docker-compose up -d db s3
+
+# We add the information we need.
+cargo run -- database migrate
+cargo run -- build crate sysinfo 0.23.4
```

Comment:
this won't work on mac systems, or on non-amd64 linux systems (from what I remember). 

But I would be fine leaving it as-is and fixing it later when I find the time :) 

If you want time with it, I typically use the web container to run builds via `docker compose run web build crate ...`

Reply:
Is having the time really that important here?

Reply:
Yes please do. I did my best at trying to set up something as easy as possible to use so you don't have to install `node`, `npm` and `npm` packages but I'm not sure my dockerfile is good. ^^'